MD/md-n/README.md

```diff
@@ -0,0 +1,137 @@
+
+If we consider allowing forks, however, it is initially unclear how perfectness translates. Naively, our assumption that there is globally one perfect expert means that there is one **perfect** fork. 
+
+We can, however, apply a reduced criterion and state that a each **fork** be comprised of a chain of consensus rounds each of which intersect in one unit of honest voting power w.r.t. to the fork itself. We call this **fork-perfectness**.
+
+Preserving **fork-perfectness** reduces to ensuring the fraction of stake which decided the original fork maintains a supermajority. If this does not occur, then it is impossible to have the guaranteed intersection. 
+
+### A3: Awareness of Minority Decisions
+
+The requirement of **full-synchronicity** means that if a supermajority decision is not made by Global Stabilization Time, some form of minority decision must effectively be made. This does not inherently mean that a Byzantine fraction of voting power may decide the global state. For example, if long-lived forks are preserved, there are multiple global states and eventual supermajority consistency can be achieved via a reconstitution of supermajority stake on an honest fork. In other words, while a **partially-synchronous** system renders eventual liveness and guaranteed safety, a **full-synchronous** system can render guaranteed liveness and eventual safety. 
+
+Regardless, Dongmen Postconfirmation protocols are required to provide a formal model of the expected value of consensus on a Byzantine minority w.r.t. the value of the state and intrinsic rewards. In other words, they must explain the nature of eventual safety that abides by Byzantine assumptions or else describe the non-BFT nature of the protocol at full-synchronous decision points. 
+
+### A4: Example Minority-Aware Protocol
+Consider the following fully-synchronous protocol:
+
+1. Votes $v \in V$ are cast for states $s \in S$ at height $h \in H$.
+2. A decision in made on a vote $s, h$ by time $t \in T$.
+    1. If $V_{h}(s) > \frac{2*|V|}{3}$, accept $s, h$
+    2. Otherwise, begin **Play Foward** algorithm.
+
+The **Play Forward** algorithm is as follows:
+
+Given a duration $d: (t + d) \in T$, and a slot duration $l: (d/l) \in T$, for each slot $l'$ in the duration $t, t + l, ... t + d$ $h' = h + (l'/l)$. 
+
+If $l' > d$, accept the root $s, h$ of the heaviest remaining subtree by weight. 
+
+Otherwise, for each tuple $(s, h') \in V_{h'}$ 
+
+1. If $V_{h'}(s) > \frac{2*|V|}{3}$, accept $s, h$, i.e., the original commitment at the root of the subtree. 
+2. If $V_{h'}(s) = \text{argmin} V_{h'}(s)$, remove all tuples $s, h \leq V_{h'}(s)$ and their descendants from the entire tree.
+
+At any given round, the number of ways the Byzantine fraction $\frac{|V|}{3} - 1$ can be eliminated is $n' = (incomplete)$ out of the total number of combinations $n$. 
```

Comment:
- first time of mention of round. it is not defined
- it is not N/3 -1 : the assumption is f/(3f+1),  N = 3f+1 , therefore f = (N-1)/3